CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,48 @@
+# Contributors Guide
```

Comment:
Can you please add a section in the README to add a link to this doc? :D

Reply:
sure, added.